v1-2/learn/how-to-use-java-interoperability.md

```diff
@@ -0,0 +1,939 @@
+---
+layout: ballerina-inner-page
+title: How to Use Java Interoperability
+permalink: /v1-2/learn/how-to-use-java-interoperability/
+---
+
+# How to call Java code from Ballerina
+
```

Comment:
Better to add a TOC as this Guide is lengthy.

Reply:
TOC is available in the Overview section